pkg/apps/database/postgres/postgres.go

```diff
@@ -71,7 +71,7 @@ func (p *Postgres) GetPostgresReplicationController() (*components.ReplicationCo
 		ContainerConfig: &horizonapi.ContainerConfig{
 			Name:       p.Name,
 			Image:      p.Image,
-			PullPolicy: horizonapi.PullIfNotPresent,
+			PullPolicy: horizonapi.PullAlways,
```

Comment:
Why are we changing the pull policy?

Reply:
based on the link, https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/rhscl/postgresql-96-rhel7

The image will be tagged as 1 and latest. If there is security fix happened on this image, we need to pull the latest.